GliaWidgets/ViewController/Survey/Components/ButtonView/Survey.ButtonView.swift

```diff
@@ -62,10 +62,10 @@ extension Survey {
         override func defineLayout() {
             super.defineLayout()
             NSLayoutConstraint.activate([
-                value.topAnchor.constraint(equalTo: topAnchor, constant: 1),
-                value.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 1),
-                value.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 1),
-                value.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 1)
+                value.topAnchor.constraint(equalTo: topAnchor, constant: 0),
```

Comment:
Do we need to remove 1pt paddings?

Reply:
Yes, it made the offset visible below border